extensions/jackson/deployment/src/main/java/io/quarkus/jackson/deployment/JacksonProcessor.java

```diff
@@ -100,6 +103,12 @@ public class JacksonProcessor {
     private static final String PARAMETER_NAMES_MODULE = "com.fasterxml.jackson.module.paramnames.ParameterNamesModule";
     private static final DotName JACKSON_MIXIN = DotName.createSimple(JacksonMixin.class.getName());
 
+    private static final ClassDesc OBJECT_MAPPER_CLASS_DESC = ClassDesc.of(ObjectMapper.class.getName());
```

Comment:
`Reflection2Gizmo.classDescOf(ObjectMapper.class)` (with a `static` import of the method) is a little bit shorter.